plugins/processors/scale/sample.conf

```
@@ -0,0 +1,27 @@
+# Scale values with a predefined range to a different output range.
+[[processors.scale]]
+    ## It is possible to define multiple different scaling that can be applied
+    ## do different sets of fields. Each scaling expects the following
+    ## arguments:
+    ##   - input_minimum: Minimum expected input value
+    ##   - input_maximum: Maximum expected input value
+    ##   - output_minimum: Minimum desired output value
+    ##   - output_maximum: Maximum desired output value
+    ##   - fields: a list of field names (or filters) to apply this scaling to
+    
+    ## Example: Define a scaling
+    # [processors.scale.scaling]
+    #    input_minimum = 0
+    #    input_maximum = 1
+    #    output_minimum = 0
+    #    output_maximum = 100
+    #    fields = ["temperature1", "temperature2"]
+    
+    ## Multiple scalings can be defined simoultaniously
+    ## Example: A second scaling. 
+    # [processors.scale.scaling]
+    #    input_minimum = -2800
+    #    input_maximum = 100
+    #    output_minimum = -20
+    #    output_maximum = 40
+    #    fields = ["humidity1", "humidity2"]
```

Maybe also show an example of the glob filtering? (I'm not sure this is the correct syntax)
```suggestion
    #    fields = ["humidity*"]
```